web/src/App.tsx

```diff
@@ -142,16 +158,7 @@ const App = () => {
             <Header />
             <main className="main grow">
                 <Routes>
-                    <Route
-                        path="/"
-                        element={
-                            process.env.REACT_APP_ISDOCKEREXT === 'true' ? (
-                                <SigninPage />
-                            ) : (
-                                <Navigate to="onboarding" replace />
-                            )
-                        }
-                    />
+                    <Route path="/" element={<OnboardingPage />} />
```

Comment:
@rymkapro should I revert to `<Navigate to="onboarding" replace />`?

Reply:
would be better, yes